sherlock_project/resources/data.json

```
@@ -1459,6 +1465,12 @@
     "urlMain": "https://www.native-instruments.com/forum/",
     "username_claimed": "jambert"
   },
+  "namuwiki": {
+    "errorType": "status_code",
+    "url": "https://namu.wiki/w/%EC%82%AC%EC%9A%A9%EC%9E%90:{}",
+    "urlMain": "https://namu.wiki/",
+    "username_claimed": "namu"
+  },
```

Can't hurt to add a note for later contributors. Prevents accidental removal by people thinking it's broken.

```suggestion
  "namuwiki": {
    "__comment__": "This is a Korean site and it's expected to return false negatives in certain other regions.",
    "errorType": "status_code",
    "url": "https://namu.wiki/w/%EC%82%AC%EC%9A%A9%EC%9E%90:{}",
    "urlMain": "https://namu.wiki/",
    "username_claimed": "namu"
  },
```